### PR TITLE
[frontend] Disable URL redirection in Markdowns (#3403)

### PR DESCRIPTION
**Pull request by Archidoit**
Github issue:
https://github.com/OpenCTI-Platform/opencti/issues/3403

Notion page:
https://www.notion.so/filigran/Disable-URL-redirection-in-Markdowns-3403-1c197ebb6790432480f65a9567a498fc

### CODE REVIEW

**Review by SarahBocognano — DISMISSED** [review]

**Comment by SarahBocognano** [comment]
Tested ✅

**Comment by richard-julien** [comment]
We also need to deactivate the links in html extended editor @Archidoit 
See HTML Files on content tab.
Moving to draft until fix

**Comment by Archidoit** [comment]
This PR is approved but NEEDS A SECOND REVIEW !!!!

**Pull request merged**
